mafia-react/src/pages/GameRoom.tsx

```diff
@@ -121,7 +124,7 @@ export const GameRoom = (props: Props) => {
         <div>
           <EntryModal
             isRoomMaster={data?.data.roomMaster === Cookies.get('userId') ? true : false}
-            playerData={playerData}
+            playerDataExists={playerData !== undefined ? true : false}
```

Comment:
can be simplified (already returns a boolean):
```suggestion
            playerDataExists={playerData !== undefined}
```